README.md

```diff
@@ -721,6 +722,20 @@ black statick statick_tool tests
 docformatter -i --wrap-summaries 88 --wrap-descriptions 88 <file>
 ```
 
+## Third Party Plugins
```

Comment:
some of these plugins are not from third parties, maybe `External Plugins` would be more accurate?